### PR TITLE
Move App component to components directory

### LINKED ISSUE #16: Moving App to components directory

### PR DESCRIPTION
**Pull request by aonomike**
fixes #16

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import ArtistContainer from './ArtistContainer'
+const App = () => {
+    return <div><ArtistContainer /></div>
+  };
+
+export default App
```
Comment (FedericoEsparza) [inline src/components/App.js]: @aonomike The `Artist` container won't be placed directly into the `App` component.
 
Instead, use routing within `App` (`react-router-dom`), to provide a route for it.
See Stephen Grider's lectures 194 and forward or the following lecture on Links in Routes:
https://www.udemy.com/react-redux/learn/v4/t/lecture/12700571?start=0
 
You'll want to use something like this for the Artist:
`<Route path="/artists/:id" component={ArtistContainer} />`

**Review by FedericoEsparza — CHANGES_REQUESTED** [review]
@aonomike Please run `git pull` and `yarn`, as I've updated the branch with some changes from `develop` to `package.json`.
 
You'll probably also need to run `nvm install v10.13.0`

File: src/components/App.js (hunk 1 above)
Comment (javpet) [reply src/components/App.js]: Sure! So the nav stays outside in the app right where we route the rest of the links?

Comment (FedericoEsparza) [reply src/components/App.js]: @javpet Yep. It can be a sibling component to the `<Route />` components.

Comment (javpet) [reply src/components/App.js]: Started the progress here: https://github.com/AgileVentures/sfn-client/pull/27

**Review by FedericoEsparza — APPROVED** [review]
@aonomike @javpet Removed App.js in its current form, since #27 is doing that work with routing.

**Pull request merged**